app/victoria-metrics/main.go

```diff
@@ -18,6 +20,15 @@ var httpListenAddr = flag.String("httpListenAddr", ":8428", "TCP address to list
 
 func main() {
 	flag.Parse()
+	reSep := regexp.MustCompile(`[a-zA-Z0-9_:]*`)
```

Comment:
This check isn't needed, since it may break setups with `.` as separator or other non-standard separators.